cluster-sim/cluster_sim/fake_power_control.py

```diff
@@ -242,16 +243,16 @@ def login(self):
         # telnet control characters for logging.
         username = self._strip_controls(self.fd.readline().rstrip())
         self.fd.readline()
-        log.debug("Received username: %s" % username)
+        log.debug(_("Received username: %s") % username)
         self.fd.write("\rPassword : ")
         self.fd.flush()
         password = self.fd.readline().rstrip()
         self.fd.readline()
-        log.debug("Received password: %s" % password)
+        log.debug(_("Received password: %s") % password)
         self.control_console()
 
     def control_console(self, *args):
-        banner = """\n\n
+        banner = _("""\n\n
```

Comment:
This the UI output of a simulated APC PDU.  Do you know that this PDU supports multiple languages and if so, that the translation is accurate?

Reply:
I've used some PDU/UPS from APC but never seen anything other than en_US there :)
Is there any reference to their localized manuals?

Reply:
So, until we can verify that APC PDUs are localized we should probably not try to translate the simulated UI, yes?

Additionally, we probably don't want to be testing non-localized IML code against even a simulated localized PDU i think.

Reply:
Heh... There *is* a manpage in russian...
http://www.apc.com/salestools/PMAR-8P9G2C/PMAR-8P9G2C_R2_RU.pdf

I'll check it out.

Reply:
No way.
It's untranslated.
What we have in russian manual:
![image](https://user-images.githubusercontent.com/407437/31822227-b843611c-b5b0-11e7-9642-2d6ade9bc162.png)
versus the original one:
![image](https://user-images.githubusercontent.com/407437/31822243-c49d8e56-b5b0-11e7-992b-1564a37d2582.png)

Reply:
So, we should probably remove the localization from the PDU interface simulation as well as avoiding localizing any IML code that interacts with the APC PDUs.

Reply:
Yes, I're right.
And there should be comments for translators to leave things as they are where appropriate.

Reply:
:+1: